app/models/user.ts

```diff
@@ -5,6 +5,15 @@ export class User {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Column('varchar')
-  username: string;
+  @Column({ type: 'varchar' })
+  name: string;
+
+  @Column({ type: 'varchar' })
+  lastname: string;
```

Comment:
use camelCase for variables inside javascript https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#34--code

Reply:
Sorry, I thought _lastname_ was a single word.